### PR TITLE
Updates for generating CORE IC files

### PR DESCRIPTION
**Pull request by jkrasting**
@adcroft - Could you help with makefile edits again?

These commands need to be executed to generate the merged files:

```
python merge_mon_seas.py -F -v ptemp_an woa13_decav_ptemp_monthly_01.nc woa13_decav_ptemp_seasonal_01.nc
python merge_mon_seas.py -F -v s_an woa13_decav_s_monthly_01.nc woa13_decav_s_seasonal_01.nc
```

### CODE REVIEW

**Comment by adcroft** [comment]
- The new pressure dependence requires new checksums
- I don't see anywhere that invokes the new script to create the combined monthly/seasonal files

**Comment by jkrasting** [comment]
Yes, I was hoping you could do your magic with makefiles again to invoke
the script that calls the new monthly/seasonal files.  :)

I can rerun the checksums once the plumbing is all connected.

**Comment by adcroft** [comment]
I can't take the pressure dependence without checksums. The repository would be in consistent. Update the checksums for the ptemp files so I can verify them. The susm for the combined files can be added when the Makefile invokes the script.

**Comment by jkrasting** [comment]
Gotcha.  Running everything now.

**Comment by jkrasting** [comment]
The checksums are updated now. See 3ac5a5a.

File: merge_mon_seas.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    nbounds =  dataset.createDimension('nbounds', 2)
+    # Copy required dimensions from source netcdf file
+    for d in field.dimensions[1::]:
+        dimension = f_seas[d]
+        dim = dataset.createDimension(d, len(dimension))
+        dims = dataset.createVariable(d, np.float64, (d,))
+        atts = dimension.__dict__
+        # Rename axis attribute to cartesian_axis 
+        if 'axis' in atts.keys():
+            atts['cartesian_axis'] = atts.pop('axis')
```
Comment (adcroft) [inline merge_mon_seas.py:73]: What is this renaming about?

**Comment by jkrasting** [comment]
Most of our input datasets have "cartesian_axis" rather than "axis".

**Pull request merged**